vault/token_store.go

```diff
@@ -1138,6 +1151,21 @@ func (ts *TokenStore) handleCreateCommon(
 			}
 		}
 
+	case role != nil && len(role.DisallowedPolicies) > 0:
+		if len(data.Policies) == 0 {
+			data.Policies = parent.Policies
+		}
+		sanitizedInputPolicies := policyutil.SanitizePolicies(data.Policies, true)
+
+		// Do not voluntarily add 'default' to the list of items to check on
```

Comment:
One thing to check -- if `default` is part of the disallowed policies, will the SanitizePolicies call strip it out? Or will it simply not add in? We do want them to be able to disallow `default`.